### PR TITLE
Added playbook contribution guide

### PR DESCRIPTION
**Pull request by idovandijk**
## Status
Ready

## Related Issues
related to: https://github.com/demisto/etc/issues/17791

## Description
Added a simplified playbook contribution guide for external and internal contributors.
View it here: https://github.com/demisto/content/blob/26dffa9e83613b0242474fae7930a429f9bf4272/docs/playbook_contribution_guide/README.md

### CODE REVIEW

File: docs/playbook_contribution_guide/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+## Exporting playbooks  
+- Your playbooks contain playbook and task descriptions by now and they should be able to run smoothly.  
+- In order to contribute your newly created playbooks, they have to be exported via the "Export" button in playbook view mode:  
+![image](https://user-images.githubusercontent.com/43602124/69058801-07d5c180-0a1d-11ea-8bd0-9dfd874b51b5.png)  
+ - The playbook will be exported as a YML file. Run the script `update_playbook.py` against the YML file. The script is located in `content\Utils\update_playbook.py`. The script will modify some fields in the file to normalize it with the rest of the playbooks in our content, and will output a file with the prefix `playbook-` in the filename. That is the file you have to use from now on.
```
Comment (yaakovi) [inline docs/playbook_contribution_guide/README.md]: switch to the sdk command

File: docs/playbook_contribution_guide/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+
+# Playbook Contribution Guide  
+## Introduction  
+This guide is intended to guide you through the process of contributing playbooks to our content, after they were created through the user interface.   
+  
+## Guidelines
+* For general guidelines on **how to create playbooks**, visit our [Creating Playbooks](https://github.com/demisto/content/tree/master/docs/creating_playbooks) article.
```
Comment (yaakovi) [inline docs/playbook_contribution_guide/README.md]: update link to the docs site

File: docs/playbook_contribution_guide/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+Examples of parent playbooks can be `Phishing Investigation - Generic v2`, or `Endpoint Malware Investigation - Generic` because an incident starts with them. 
+Examples of sub-playbooks are `IP Enrichment - Generic v2` or `Retrieve File From Endpoint - Generic`, because they are steps we take as part of the bigger investigation.
+What one needs to consider is that since sub-playbooks are used as part of a bigger investigation, **they should have inputs and outputs.** Make sure that data you want to get from a sub-playbook, is defined in the outputs of it, so it can be used outside of that playbook. Since sub-playbooks are building blocks that will preferably be usable in other playbooks and use-cases, you should define **generic inputs** for them as explained in our [context standards](https://github.com/demisto/content/tree/master/docs/context_standards) article.
```
Comment (yaakovi) [inline docs/playbook_contribution_guide/README.md]: same here

**Comment by glicht** [comment]
@idovandijk please close this and re-open in: https://github.com/demisto/content-docs

File: docs/playbook_contribution_guide/README.md (hunk 1 above)
Comment (idovandijk) [reply docs/playbook_contribution_guide/README.md]: What is the command?

**Pull request closed without merging**